apps/storybook.namekit.io/stories/Namekit/DisplayedPrice.stories.tsx

```diff
@@ -40,654 +86,201 @@ export default meta;
-      </span>
-    ),
-  },
-};
-export const UsdMaxDisplayPriceWithAcronym: Story = {
-  args: {
-    price: {
-      currency: Currency.Usd,
-      value: 1000000000000000000n,
-    },
-    symbol: (
-      <span>
-        {getCurrencySymbology(Currency.Eth, CurrencySymbology.Acronym)}
-      </span>
-    ),
-  },
-};
-export const UsdcMinDisplayPriceWithAcronym: Story = {
-  args: {
-    price: {
-      currency: Currency.Usdc,
-      value: 1n,
-    },
-    symbol: (
-      <span>
-        {getCurrencySymbology(Currency.Eth, CurrencySymbology.Acronym)}
-      </span>
-    ),
-  },
-};
-export const UsdcMaxDisplayPriceWithAcronym: Story = {
-  args: {
-    price: {
-      currency: Currency.Usdc,
-      value: 1000000000000000000n,
-    },
-    symbol: (
-      <span>
-        {getCurrencySymbology(Currency.Eth, CurrencySymbology.Acronym)}
-      </span>
-    ),
-  },
-};
-export const DaiMinDisplayPriceWithAcronym: Story = {
-  args: {
-    price: {
-      currency: Currency.Dai,
-      value: 1n,
-    },
-    symbol: (
-      <span>
-        {getCurrencySymbology(Currency.Eth, CurrencySymbology.Acronym)}
-      </span>
-    ),
-  },
-};
-export const DaiMaxDisplayPriceWithAcronym: Story = {
-  args: {
-    price: {
-      currency: Currency.Dai,
-      value: 10000000000000000000000000n,
-    },
-    symbol: (
-      <span>
-        {getCurrencySymbology(Currency.Eth, CurrencySymbology.Acronym)}
-      </span>
-    ),
-  },
-};
-export const WethMinDisplayPriceWithAcronym: Story = {
-  args: {
-    price: {
-      currency: Currency.Weth,
-      value: 1n,
-    },
-    symbol: (
-      <span>
-        {getCurrencySymbology(Currency.Eth, CurrencySymbology.Acronym)}
-      </span>
-    ),
-  },
-};
-export const WethMaxDisplayPriceWithAcronym: Story = {
-  args: {
-    price: {
-      currency: Currency.Weth,
-      value: 1000000000000000000000000n,
-    },
-    symbol: (
-      <span>
-        {getCurrencySymbology(Currency.Eth, CurrencySymbology.Acronym)}
-      </span>
-    ),
-  },
-};
-export const EthMinDisplayPriceWithSymbol: Story = {
-  args: {
-    price: {
-      currency: Currency.Eth,
-      value: 100000000000000n,
-    },
+    price: numberAsPrice(0.001, Currency.Eth),
     symbol: (
       <CurrencySymbol
         currency={Currency.Eth}
-        describeCurrencyInTooltip={true}
+        symbology={CurrencySymbology.Icon}
       />
     ),
   },
 };
-export const EthMaxDisplayPriceWithSymbol: Story = {
+export const EthMaxDisplayPriceWithCustomCurrencyIcon: Story = {
   args: {
-    price: {
-      currency: Currency.Eth,
-      value: 10000000000000000000000000n,
-    },
+    price: numberAsPrice(1000000, Currency.Eth),
     symbol: (
       <CurrencySymbol
         currency={Currency.Eth}
-        describeCurrencyInTooltip={true}
+        symbology={CurrencySymbology.Icon}
       />
     ),
   },
 };
-export const UsdMinDisplayPriceWithSymbol: Story = {
+export const UsdMinDisplayPriceWithCustomCurrencyIcon: Story = {
```

Comment:
Hey, no big idea behind this. Goal was that, since we have a lot of Currency possibilities, to display how to use our components for building each Currency displaying price. I find it good to make it summarized for one or two examples, people might get the idea when they notice that CurrencySymbol has support to more currencies